Modix.Data.Test/TestData/Tags/ReadSummaryAsync/ExceptionReadTestData.cs

```diff
@@ -4,6 +4,6 @@ namespace Modix.Data.Test.TestData.Tags.ReadSummaryAsync
 {
     internal class ExceptionReadTestData : ReadTestDataBase
     {
-        public Type ExceptionType { get; set; }
+        public Type ExceptionType { get; set; } = null!;
```

Comment:
Personally, I think that looks uglier than the switch expressions in the Abstractions library. It's your code so I can change it if you really prefer, but this is much more readable imo.

Reply:
"Nitpick" means you are free to leave it as-is.

I prefer newlines here because A) I generally prefer one "logical thought" per-line, and to me defining the property and initializing its value are separate thoughts, and B) It makes the "definition" line more consistent with the other property definitions on the same class, which don't require initialization.